frontend/lib/framework/Context.cpp

```diff
@@ -88,9 +92,16 @@ llvm::ErrorOr<const ChplEnvMap&> Context::getChplEnv() {
 }
 
 void Context::defaultReportError(Context* context, const ErrorBase* err) {
-  ErrorWriter ew(context, std::cerr, ErrorWriter::BRIEF,
+  ErrorWriter ew(context, std::cerr,
+                 context->detailedErrors ?
+                   ErrorWriter::DETAILED :
+                   ErrorWriter::BRIEF,
                  context->currentTerminalSupportsColor_);
   err->write(ew);
+  if (context->detailedErrors) {
+    // Print an extra error separator
+    std::cerr << std::endl;
```

Comment:
Someone earlier pointed out that the detailed errors feel too cramped (the new error's line starts right after another error's text). So, I print a newline :)

Reply:
Makes sense. I was thinking about how all the detailed .good files have an extra newline at the end I would like to avoid, but I doubt there's any good way to do that and also keep the extra spacing between multiple detailed errors.